src/web/components/TeamMember/TeamMember.tsx

```diff
@@ -10,29 +10,26 @@ import { InlineMessage } from '../Core/InlineMessage';
 import { SuccessToast } from '../Core/Toast';
 import TeamMemberDialog from './TeamMemberDialog';
 
-type DeleteConfirmationDialogProps = {
+type DeleteConfirmationDialogProps = Readonly<{
   person: UserResponse;
   onRemoveTeamMember: () => Promise<void>;
-};
-
-function DeleteConfirmationDialog({ person, onRemoveTeamMember }: DeleteConfirmationDialogProps) {
-  const [openConfirmation, setOpenConfirmation] = useState(false);
+  onOpenChange: () => void;
+}>;
 
+function DeleteConfirmationDialog({
```

Comment:
While we aren't exporting this component, I still think it would be good to differentiate this component from the component with the same name that is exported from `CstgDeleteDomainDialog.tsx`. Ideally, even extract to a separate file like you did for the the CSTG domains one.

What do you think? Feel free to raise as a separate ticket.

Reply:
Moved! I agree its better to be extracted and renamed.